### PR TITLE
remove field on docr and force recreate on do

### PR DESCRIPTION
**Pull request by maathor**

### CODE REVIEW

File: lib/digitalocean/bootstrap/doks_gen_kubectl-config.j2.tf [hunk 1]
```diff
@@ -17,4 +21,8 @@ resource "digitalocean_spaces_bucket_object" "upload_kubeconfig" {
   key          = "${var.oks_cluster_id}.yaml"
   source       = local_file.kubeconfig.filename
   depends_on = [local_file.kubeconfig, digitalocean_spaces_bucket.space_bucket_kubeconfig]
+
```
Comment (deimosfr) [inline lib/digitalocean/bootstrap/doks_gen_kubectl-config.j2.tf:24]: Why is this useful? It's a local file

File: src/container_registry/docr.rs [hunk 2]
```diff
@@ -56,10 +54,9 @@ impl DOCR {
 
     pub fn create_repository(&self, _image: &Image) -> Result<(), EngineError> {
         let mut headers = get_header_with_bearer(&self.api_key);
-        //TODO: receive subscription_tier_slug from core
         let repo = DO_API_Create_repository {
             name: self.registry_name.clone(),
-            subscription_tier_slug: "basic".to_owned(),
+            subscription_tier_slug: "professional".to_owned(),
```
Comment (deimosfr) [inline src/container_registry/docr.rs:59]: What is this?

Comment (maathor) [reply src/container_registry/docr.rs:61]: ok i added some comment ;)

File: lib/digitalocean/bootstrap/doks_gen_kubectl-config.j2.tf (hunk 1 above)
Comment (maathor) [reply lib/digitalocean/bootstrap/doks_gen_kubectl-config.j2.tf]: you're right

**Review by deimosfr — APPROVED** [review]

**Pull request merged**